src/tests/cases/ui.rs

```diff
@@ -583,6 +583,47 @@ fn sustained_traffic_from_one_process() {
     assert_snapshot!(&terminal_draw_events_mirror[2]);
 }
 
+#[test]
+fn sustained_traffic_from_one_process_total() {
+    let network_frames = vec![NetworkFrames::new(vec![
+        Some(build_tcp_packet(
+            "1.1.1.1",
+            "10.0.0.2",
+            12345,
+            443,
+            b"I have come from 1.1.1.1",
+        )),
+        None, // sleep
+        Some(build_tcp_packet(
+            "1.1.1.1",
+            "10.0.0.2",
+            12345,
+            443,
+            b"Same here, but one second later",
+        )),
+    ]) as Box<dyn DataLinkReceiver>];
+
+    let (terminal_events, terminal_draw_events, backend) = test_backend_factory(190, 50);
+
+    let os_input = os_input_output(network_frames, 3);
+    let mut opts = opts_ui();
```

Comment:
I think these still use `total_utilization: false`, don't they?

Reply:
Initially, `opts_ui()` does have `total_utilization` set to false, but I'm setting it to true in the line immediately after. Adding another "opts building" function was a bit messy, so I opted for that. I could define an `opts_ui_total()` function if you'd rather.

The new snapshots from these tests do show totals as opposed to rates ("B" not "Bps"). Perhaps I'm missing something?

Reply:
Oh man, I managed to totally miss that line! My bad, sorry. :)

Reply:
Haha, no worries :)